packages/hawtio/src/plugins/shared/components/attributes/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Attributes } from './Attributes'
```

Comment:
If we move `attributes` to `shared`, I wouldn't like to move it to `shared/components` for the same reason as features over functions. Can we move it to `shared/attributes`?